pom.xml

```
@@ -11,10 +11,10 @@
     <parent>
 		<groupId>life.qbic</groupId>
 		<artifactId>cli-parent-pom</artifactId>
-		<version>2.1.0</version>
+		<version>3.1.4</version>
 	</parent>
 	<artifactId>sample-status-updater-cli</artifactId>
-	<version>1.3.0-SNAPSHOT</version>
+	<version>1.2.4</version>
```

```suggestion
	<version>1.3.0-SNAPSHOT</version>
```
We need to stay with the current version